.devcontainer/devcontainer.json

```diff
@@ -5,16 +5,22 @@
     "../docker-compose.yml"
   ],
 
-  "service": "embedded-sharepoint",
+  "service": "dev",
   "workspaceFolder": "/Embedded-Sharepoint",
   "shutdownAction": "stopCompose",
 
-  "settings": {
-    "terminal.integrated.defaultProfile.linux": "bash",
-    "editor.formatOnSave": true
-  },
-
-  "extensions": [
-  ]
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "terminal.integrated.defaultProfile.linux": "bash"
+      },
+      "extensions": [
+        "ms-vscode.cpptools-extension-pack",
+        "ms-python.python",
+        "GitHub.vscode-pull-request-github",
+        "ms-vscode.makefile-tools"
```

Comment:
Maybe gitlens should be added as a required extension...community edition is pretty feature rich and can make git debug a bit easier.

Up 2 yall tho